src/ape/cli/commands.py

```diff
@@ -0,0 +1,20 @@
+from typing import Any
+
+import click
+from click import Context
+
+from ape import networks
+
+
+def network_bound_command(*args, **kwargs):
```

Comment:
Past tense, would `bind_network_command` be a better name?

Reply:
Better: `command_with_network_option` now would make a lot more sense

Reply:
What about something like `command_using_network_option`? That way it still conveys that the network option get used upon invocation

Reply:
yes! and then me (a plugin writer) parsing this out:
"oh, okay this makes sense, if I add `network` to my args I get access to the network option that `command_using_network_option` provides!"